java/src/jmri/jmrix/openlcb/swing/downloader/LoaderPane.java

```diff
@@ -136,6 +143,22 @@ protected void handleOptionsInFileContent(MemoryContents inputContent) {
     @Override
     protected void doLoad() {
         super.doLoad();
+        
+        // de-cache CDI information so next window opening will reload
+        iface.dropConfigForNode(destNodeID());
+
+        // if window referencing this node is open, close it
+        var frames = jmri.util.JmriJFrame.getFrames();
+        for (var frame : frames) {
+            if (frame instanceof NodeSpecificFrame) {
+                if ( ((NodeSpecificFrame)frame).getNodeID() == destNodeID() ) {
+                    // This window references the node and should be closed
+                    frame.dispatchEvent(new WindowEvent(frame, WindowEvent.WINDOW_CLOSING));
```

Comment:
No need to change, but a thought.

Could the user be confused if a window suddenly disappears? I haven't used OpenLCB so I don't know how this works, but I'm thinking of if it could be good to have some information to the user on why the window has closed. Either a message box telling that some windows has been closed or a message in the log.